src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -21,7 +21,7 @@ function checkFeedbackQuestion(form) {
         setStatusMessage(DISPLAY_FEEDBACK_QUESTION_TEXTINVALID, StatusType.DANGER);
         return false;
     }
-    if ($(form).find('[name=' + FEEDBACK_QUESTION_TYPE + ']').val() === 'NUMSCALE') {
+    if ($(form).find('[id=' + FEEDBACK_QUESTION_TYPE + ']').val() === 'NUMSCALE') {
```

Comment:
I didn't think about that, validation won't work for editing old questions this way. I reseted this back to 'name'.